doc/source/qip-qiskit.rst

```diff
@@ -0,0 +1,116 @@
+******************************
+Qiskit
+******************************
+
+Overview
+===============
+
+This submodule provides an interface to simulate circuits made in qiskit.
+
+Gate-level simulation on qiskit circuits is possible with :class:`.QiskitCircuitSimulator`. Pulse-level simulation is possible with :class:`.QiskitPulseSimulator` which supports simulation using the :class:`.LinearSpinChain`, :class:`.CircularSpinChain` and :class:`.DispersiveCavityQED` pulse processors.
+
+Running a qiskit circuit with qutip_qip
+==========================================
+
+After constructing a circuit in qiskit, either of the qutip_qip based backends (:class:`.QiskitCircuitSimulator` and :class:`.QiskitPulseSimulator`) can be used to run that circuit.
+
+Example
+--------
+
+Let's try constructing and simulating a qiskit circuit.
+
+We define a simple circuit as follows:
+
+.. code-block::
+
+    from qiskit import QuantumCircuit
+    
+    circ = QuantumCircuit(2,2)
+    circ.h(0)
+    circ.h(1)
+    
+    circ.measure(0,0)
+    circ.measure(1,1)
+
+Let's run this on the :class:`.QiskitCircuitSimulator` backend:
+
+.. code-block::
+
+    from qutip_qip.qiskit.provider import QiskitCircuitSimulator
+
+    backend = QiskitCircuitSimulator()
+    job = backend.run(circ)
+    result = job.result()
+    
+>>> from qiskit.visualization import plot_histogram
+>>> plot_histogram(result.get_counts())
+
+.. image:: /figures/qiskit-gate-level-plot.png
+    :alt: probabilities plot 
+
+Now, let's run the same circuit on :class:`.QiskitPulseSimulator`.
+
+While using a pulse processor, we define the circuit without measurements:
+
+.. code-block:: 
```

Comment:
It would be nice to use `doctest` or `testcode` as much as possible because they will be executed when testing the doc build, while `code-block` won't.

You can also use [`testsetup`](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html) to predefine some importation.